src/utils.js

```diff
@@ -12,15 +12,7 @@ export function domReady() {
 }
 
 export function arrayUnique(array) {
-    var a = array.concat();
-    for(var i=0; i<a.length; ++i) {
-        for(var j=i+1; j<a.length; ++j) {
-            if(a[i] === a[j])
-                a.splice(j--, 1);
-        }
-    }
-
-    return a;
+    return Array.from(new Set(array))
```

Comment:
this probably doesn't work in IE11. Please see the thread.

Reply:
I didn't really pay attention on that. I'll update PR with the thread suggestion. Thanks!

Reply:
I'm not sure about not using the `new Set(array)` syntax. 
There is more code non-compatible to IE11. e.g. all the usage of [spread syntax](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_syntax). <br>
- [here](https://github.com/alpinejs/alpine/blob/master/src/utils.js#L67)
- [here](https://github.com/alpinejs/alpine/blob/master/src/utils.js#L68) 

or [Array.includes](https://github.com/alpinejs/alpine/blob/master/src/utils.js#L127), which is also not supported in IE11
- [here](https://github.com/alpinejs/alpine/blob/master/src/utils.js#L127)
- [here](https://github.com/alpinejs/alpine/blob/master/src/utils.js#L151)

You may ensure compatibility for IE11 either **everywhere** or **nowhere**. 

cheers
flashe :zap:

Reply:
@flash-me
Babel convert the spread syntax when compiling but it doesn't convert `new Set(iterator)`.
If you look at the compiled IE11 code in the dist folder, you won't find any trace of the spread operator.

For `Array.includes`, Alpine uses a polyfill. We could probably use another polyfill for `new Set(iterator)` if ii exists but it has to be part of this PR, imo.

Since `const set = new Set(); arr.forEach(v => set.add(v)); return Array.from(set);` is already supported by IE11, I think we should opt for this syntax, to be fair.

Reply:
tbh I didn't look into the dist folder. Shame on me 😇

And the opt-out shouldn't be that different much of what the browsers will do internally.

Since all I do was to help, I'm totally OK with this. 👍🏽 

cheers 
flash :zap:

Reply:
Now that you commented I realize that we were already using core-js polyfill and aparently It do the job about Set. I don't know how Alpine is using core-js but maybe we already can use new Set(array)

Reply:
@julioocamargoo If the core-js supports it, we are good. You don't need to add anything else, rollup should already include the polyfill for you.

Reply:
I verified and new Set([]) works fine on IE with core-js. I've never used rollup but looking in the docs and [rollup-ie11.config.js](https://github.com/alpinejs/alpine/blob/master/rollup-ie11.config.js) I think that problem was solved as @SimoTod said, so I'll mantain this code.